docs/specs/openapi.yaml

```diff
@@ -167,13 +401,22 @@ components:
         status:
           type: string
           description: The text for the HTTP status code, see https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml.
-    GetChangeWait:
+    GetChanges:
       allOf:
         - $ref: '#/components/schemas/BaseResponse'
         - type: object
           properties:
             result:
-              $ref: '#/components/schemas/ChangeInfo'
+              type: array
+              items:
+                $ref: '#/components/schemas/changeInfo'    
+    GetChange:
```

Comment:
Since this name will display in the rendered output, as the response schema for several endpoints, it might be clearer to rename it to `ChangeObject`